std/experimental/allocator/common.d

```diff
@@ -264,12 +272,14 @@ unittest
 /**
 Returns `n` rounded down to a multiple of alignment, which must be a power of 2.
 */
+@safe @nogc nothrow pure
 package size_t roundDownToAlignment(size_t n, uint alignment)
```

Comment:
Remainder to me or if someone else wants to do it:

use  `roundDownToAlignment()` in win32 version of `_aligned_malloc ()`, just [here](https://github.com/D-Programming-Language/phobos/blob/master/std/experimental/allocator/mallocator.d#L131)